misc/helm-charts/operator/templates/deployment.yaml

```diff
@@ -79,6 +79,9 @@ spec:
         {{- if not .Values.operator.features.authentication }}
         - "--disable-authentication"
         {{- end }}
+        {{- if not .Values.operator.features.databaseNetworkPolicies}}
+        - "--disable-database-network-policies"
```

Comment:
We're now flopping back and forth between positive and negative args. Can we get consistent?

Reply:
Rather than having this be a negative all the way through, which was also commented on as being less than ideal, I'm trying to follow the `--diable-authentication` flag which does this flopping.